src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+#if USE_MDT_EVENTSOURCE
+using Microsoft.Diagnostics.Tracing;
+#else
+using System.Diagnostics.Tracing;
+#endif
+using Xunit;
+
+using SdtEventSources;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.Diagnostics.Tracing.Session;
+using Microsoft.DotNet.RemoteExecutor;
+using Microsoft.Diagnostics.Tracing;
+
+namespace BasicEventSourceTests
+{
+    public partial class TestsManifestGeneration
+    {
+
+        [Fact]
+        public void Test_EventSource_EtwManifestGeneration()
+        {
+            RemoteExecutor.Invoke(() =>
+            {
+                using (RemoteInvokeHandle handle = RemoteExecutor.Invoke(() =>
+                {
+                    var es = new SimpleEventSource();
+                    for (var i = 0; i < 100; i++)
+                    {
+                        es.WriteSimpleInt(i);
+                        Thread.Sleep(100);
+                    }
+                }))
```

Comment:
I'm not sure this is doing what you intend it to.  The code inside the lambda here won't run until `dispose` is called I think, which won't happen until after you hit the end of the `using` block and all your collection code has executed.  That would explain why the test is failing.